sql/sqlserver/server.go

```diff
@@ -256,13 +258,14 @@ func (s *Server) ShowIndex(session *Session, p *parser.ShowIndex, args []sqlwire
 	// TODO(pmattis): This output doesn't match up with MySQL. Should it?
 	var rows []sqlwire.Result_Row
 
+	name := p.Table[len(p.Table)-1]
```

Comment:
Definitely. I originally started without the `QualifiedName.{Database,Table}()` accessors, but the code was really confusing. Missed this one place.